include/quicly/cc.h

```diff
@@ -0,0 +1,38 @@
+/* Interface definition for quicly's congestion controller.
+ */
+
+#ifndef quicly_cc_h
+#define quicly_cc_h
+
+#include "quicly/constants.h"
+#include <assert.h>
+#include <stdint.h>
+#include <string.h>
+
+struct ccstate {
+    uint32_t cwnd;
+    uint32_t ssthresh;
+    uint32_t stash;
+    uint8_t recovery_end;
+};
+
+void cc_init2(struct ccstate *ccs);
```

Comment:
Do we need `2`?

Also, the discussion about prefixes and typenames applies to these functions.

Reply:
Leftover, from when cc_init was part of the dcc/ controller.